<launch>
    <!--node pkg="tf2_ros" type="static_transform_publisher" name="BRov" args="0 0 0 0.707 0 0 0.707 world Rov_frame"/-->
    <!--node pkg="tf2_ros" type="static_transform_publisher" name="BRov" args="0 0 0 0 0 0.7071068 0.7071068 world Rov_frame"/--><!-- rot around z axes 90 deg-->
    <!--node pkg="tf2_ros" type="static_transform_publisher" name="BRov" args="0 0 0 1 0 0 0 world Rov_frame"/180 rotation around x axes-->
    <node pkg="tf2_ros" type="static_transform_publisher" name="pixhawk" args="-0.023 0.020 0.079 0 0 0 1 Rov_frame pixhawk"/>
    <!--node pkg="tf2_ros" type="static_transform_publisher" name="imu" args="0 0 0 0 0 0 1 pixhawk_frame pixhawk"/-->
    <node pkg="tf2_ros" type="static_transform_publisher" name="camera" args="0.017 0.003 0.0 0 0 0 1 Rov_frame gimball"/>
    <!--node pkg="tf" type="static_transform_publisher" name="raspberry" args="-0.03 0.18 0.58 0 0 0 4 Rov_frame raspi_frame 1000"/>
    <node pkg="tf" type="static_transform_publisher" name="currentLeakSensor" args="0.13 0.00847 0.32 0 0 0 5 Rov_frame cLeak_frame 1000"/>
    <node pkg="tf" type="static_transform_publisher" name="HallSensor" args="0.20 -0.07 -0.034 0 0 0 6 Rov_frame hall_frame 1000"/-->  
    <node pkg="tf2_ros" type="static_transform_publisher" name="Bar30" args="0.016 0.006 -0.06 0 0 0 1 Rov_frame Bar30_frame"/>
       
    <!--node pkg="tf" type="static_transform_publisher" name="BRov" args="1.54 1.53 -1.29 0 1.5 0 1 world Rov_frame 1000"/>
    <node pkg="tf" type="static_transform_publisher" name="pixhawk" args="-0.23 0.20 0.79 0 0 0 3 Rov_frame pixhawk_frame 1000"/-->
    <node pkg="tf2_ros" type="static_transform_publisher" name="yellow_patch" args="0 0 0 0 0 0 1 Rov_frame yellow_patch"/>
</launch>
